sonora_eng_DLC/text/english/game/pro_scen.msg

```diff
@@ -1780,11 +1780,11 @@
 {92900}{}{Bed}
 {92901}{}{A very comfortable looking bed.}
 {93000}{}{Junk}
-{93001}{}{A bed mattress in terrible condition. Even the rats won't sleep in or on it.}
+{93001}{}{A mattress in terrible condition. Even the rats won't sleep in or on it.}

```

Comment:
You'd almost never say "bed mattress" in English because there aren't other kinds of mattresses.